src/hooks/UseQuery.re

```diff
@@ -37,11 +41,15 @@ let useQuery =
     (
       ~request,
       ~pause=?,
+      ~additionalTypenames=?,
```

Comment:
Since our hooks accept `Partial<OperationContext>` as a parameter in TS, we support the same here but with each key of `OperationContext` as a separate labelled argument.